docs/en/metadata.yaml

```diff
@@ -119,8 +119,8 @@ navigation:
     - location: utilities/vertically-center.md
       title: Vertically center
 
-    - location: utilities/visibility.md
-      title: Visibility
+    - location: utilities/show.md
```

Comment:
Sorry, should have mentioned. It a drive-by fix. The pattern name was changed but the navigation wasn't.